File: WebApp/WebApp/Models/BindingModels/CatalogInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models.BindingModels
{
    public class CatalogInfo
    {
        public IEnumerable<TicketType> TicketTypes { get; set; }
        public IEnumerable<PassengerType> PassengerTypes { get; set; }

        public IEnumerable<TicketPrices> TicketPrices { get; set; }
    }
}

File: FrontEndProjectWeb2/src/app/models/CatalogInfo.ts
import { TicketType } from './TicketType';
import { PassengerType } from './PassengerType';
import { TicketPrices } from './TicketPrices';

export class CatalogInfo {
    TicketTypes: TicketType[];
    PassengerTypes: PassengerType[];
    TicketPrices: TicketPrices[];
  }